scripts/timesTable.js

```diff
@@ -0,0 +1,14 @@
+function timesTable(rows) {
```

Comment:
this is pretty hard to follow (probably given your choice of variable names). could you be more specific with this? could you also leverage a JS Array prototype method to help you out here?